jerry-core/api/jerry-snapshot.c

```diff
@@ -917,26 +917,27 @@ jerry_generate_snapshot (const jerry_char_t *source_p, /**< script source */
 #endif /* JERRY_SNAPSHOT_SAVE */
 } /* jerry_generate_snapshot */
 
-#if JERRY_SNAPSHOT_EXEC
 /**
- * Execute/load snapshot from specified buffer
+ * Execute snapshot from specified buffer
```

Comment:
With the new flag this will load the function, and will return a function object (if I understand correctly). So removing the load seems weird. Maybe the whole method description should be reworded. Maybe: "Executes a snapshot from as specified buffer or if <flag> is used this will load/return it as a function object.".